pkg/storaged/crypto-keyslots.jsx

```diff
@@ -303,6 +303,17 @@
                                 return install_missing_packages(data, status_callback(progress));
                         }
                     });
+            })
+            .catch(error => {
+            // Something wrong with PackageKit, maybe it is not even
+            // installed.  Let's show the error during fixing.
+                progress(null, null);
+                steps.push({
+                    title: cockpit.format(_("The $0 package must be installed."), package_name),
+                    func: progress => {
+                        return Promise.reject(cockpit.format(_("PackageKit error: $0"), error.toString()));
```

Comment:
These 5 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18698-20230424-134533-fc33405c-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)